recipes/atproto/meta.yaml

```diff
@@ -19,12 +19,12 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python >=3.8,<3.14
```

Comment:
Can you re-do this but this time add

```
{% set python_min = python_min|default("3.8") %}
```

at the top?

Reply:
thank you, trying.